compiler/src/dotty/tools/dotc/classpath/ZipAndJarFileLookupFactory.scala

```diff
@@ -10,8 +10,9 @@ import java.nio.file.attribute.{BasicFileAttributes, FileTime}
 
 import scala.annotation.tailrec
 import dotty.tools.io.{AbstractFile, ClassPath, ClassRepresentation, FileZipArchive, ManifestResources}
-import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Contexts.{Context, ctx}
 import FileUtils._
+import dotty.tools.dotc.core.Contexts.{Context, ctx}
```

Comment:
This import is a duplicate of the one two lines above it.

Reply:
Fixed, thank you!